pkg/metrics/query.go

```diff
@@ -38,7 +38,7 @@ func Query(
 	// Copy URL to avoid modifying the passed value.
 	u := new(url.URL)
 	*u = *endpoint
-	u.Path = ""
+	u.Path = (*endpoint).Path
```

Comment:
If we want this logic, we can just kill this line, but I have a feeling it was needed for something... Let's find out for what 🤔

Reply:
👍 we can kill this line. So far, I could find any use for this line (for blank URL). Another data point is that all the current deployments (at stage, prod), the endpoint URLs do not have URLPath associated with them.